HeterogeneousCore/CUDAServices/plugins/BuildFile.xml

```diff
@@ -1,19 +1,19 @@
-#Skip building plugins by dropping all files for none-AMD64 build
+<!-- Skip building plugins by dropping all files for none-AMD64 build -->
 <architecture match="!_amd64_">
   <flags SKIP_FILES="*"/>
 </architecture>
-<use   name="DataFormats/Provenance"/>
-<use   name="DataFormats/Common"/>
-<use   name="DataFormats/Streamer"/>
-<use   name="FWCore/MessageLogger"/>
-<use   name="FWCore/ParameterSet"/>
-<use   name="FWCore/PluginManager"/>
-<use   name="FWCore/ServiceRegistry"/>
-<use   name="FWCore/Utilities"/>
-<use   name="FWCore/Concurrency"/>
-<use   name="FWCore/Framework"/>
-<use   name="HeterogeneousCore/CUDAServices"/>
-<use   name="cuda"/>
-<library   file="*.cc" name="HeterogeneousCoreCUDAServicesPlugins">
-  <flags   EDM_PLUGIN="1"/>
+<use name="cuda"/>
```

Comment:
I suppose this is needed for `NVProfilerService`? Should it be moved to `src` as well?

Reply:
Even if we don't `<use name="cuda"/>` in the plugins, the plugin library will depend on the main library, which depends on `cuda` - so we wouldn't gain anything AFAICT.

Reply:
But what about the `NVProfilerService` itself?